Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -8,9 +8,20 @@
 #include <LibJS/Runtime/GlobalObject.h>
 #include <LibJS/Runtime/RegExpConstructor.h>
 #include <LibJS/Runtime/RegExpObject.h>
+#include <LibJS/Runtime/Value.h>
 
 namespace JS {
 
+void RegExpLegacyProperties::invalid()
```

Comment:
Please add a spec link and comments to this function.
And maybe rename it to something more similar to the spec version

https://github.com/tc39/proposal-regexp-legacy-features#invalidatelegacyregexpstaticproperties--c